Stratis.Bitcoin.Features.Notifications/TransactionReceiver.cs

```diff
@@ -32,20 +32,20 @@ public TransactionReceiver(TransactionNotification transactionNotification, Tran
 
         protected override void AttachCore()
         {
-            this.AttachedNode.MessageReceived += this.AttachedNode_MessageReceived;
+            this.AttachedNode.MessageReceived += this.AttachedNode_MessageReceivedAsync;
         }
 
         protected override void DetachCore()
         {
-            this.AttachedNode.MessageReceived -= this.AttachedNode_MessageReceived;
+            this.AttachedNode.MessageReceived -= this.AttachedNode_MessageReceivedAsync;
         }
 
-        private async void AttachedNode_MessageReceived(Node node, IncomingMessage message)
+        private async void AttachedNode_MessageReceivedAsync(Node node, IncomingMessage message)
```

Comment:
here i'm confused, above we've had 2 instances of exactly this and we used pragmas
here we use suffix
so if there is nothing that prevents us from using that suffix, use it in all instances and do not introduce those pragmas
or am i missing something?

Reply:
Actually when I fixed the other ones I got a bunch of errors. I thought it's because of inheritance, so I just pragmad them in all other cases. However in this case I noticed the problem wasn't inheritance, but just a naming conflict. Anyway, fixed.

Reply:
i see, tx for explanation, looks good